plone/app/iterate/configure.zcml

```diff
@@ -70,15 +70,22 @@
       title="Working Copy Support (Iterate) [uninstall]"
       directory="profiles/uninstall"
       provides="Products.GenericSetup.interfaces.EXTENSION"
+      for="plone.base.interfaces.IMigratingPloneSiteRoot"
       />
 
-  <genericsetup:upgradeStep
-      source="*"
-      destination="121"
-      title="Reinstall plone.app.iterate"
-      description=""
-      profile="plone.app.iterate:plone.app.iterate"
-      handler="plone.app.iterate.util.upgrade_by_reinstall"
```

Comment:
@mauritsvanrees Removing this might prevent necessary changes from being made when upgrading from an old version, no?

The new way makes more sense, and I don't have a big problem with switching to it. I'm just thinking maybe we should leave the existing reinstall upgrade alone, and then write a new one from 121 to the new version.

Reply:
No. Metadata version 121 is from Plone 4.0.0. Updating to that old version is not supported in 6.0: you need to update to 5.2 in Python 2 first anyway.

Reply:
@mauritsvanrees Right, makes sense